Wed, 30 Aug 2017 16:24:27 sendSms.py[line:38] ERROR Exception<type 'exceptions.Exception'>:<urlopen error [Errno 8] nodename nor servname provided, or not known>

